### PR TITLE
Added submenu in settings for changing the symbol of multiplication

### PR DESCRIPTION
**Pull request by RedGl0w**
https://github.com/Omega-Numworks/Omega/issues/42
Added :
image
I think `always simplify` isn't the best text, and neither `sign of multiplication` is, but I don't have any idea...
When you choose a symbol, when you type in calculation, or in graph, this symbol will be written.
When you type enter, Poincaré will analyse which symbol should be the more present, according to https://github.com/Omega-Numworks/Omega/blob/2925fb2c4b8286a8046b425da5937772b41aca73/poincare/src/multiplication.cpp#L88-L109 , but you can disable this functionality by disabling `always simplify`...

### CODE REVIEW

**Comment by quentinguidee** [comment]
When I switch the symbol in the settings, my simulator crashes. Here are the logs of `make PLATFORM=simulator` : https://pastebin.com/eyUkfszW

Crash:
```bash
❯ ./epsilon.bin 
[1]    21802 segmentation fault (core dumped)  ./epsilon.bin
```

File: apps/settings/sub_menu/symbol_controller.cpp [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          break;
+        }
+      default:
+        {
+          GenericSubController::handleEvent(event);
+        }
+    }
+    preferences->setSymbolMultiplication(symbolofMultiplication);
+    autoSymbol ? preferences->SetAutoSymbol(Poincare::Preferences::AutoSymbol::True) : preferences->SetAutoSymbol(Poincare::Preferences::AutoSymbol::False);
+    m_selectableTableView.reloadData();
```
[the remark this answers is not in the document]
Comment (RedGl0w) [reply apps/settings/sub_menu/symbol_controller.cpp]: I've already done that suggestion

**Comment by quentinguidee** [comment]
I'm fixing the conflict

**Comment by quentinguidee** [comment]
Merged! Thank you!!

**Pull request merged**